### ISSUE TITLE
Installation Lab

### ISSUE THREAD

**Issue by bagusaptopo** [bagusaptopo/SEBC#1]

**Comment by mfernest** [comment]
What lab step are you currently working on?

**Comment by bagusaptopo** [comment]
I've done installing MySQL and CM, but have found error when trying to initialize database for CM using this command: <code>./scm_prepare_database.sh mysql cloudera_mgr root</code>

The error is:
<code>[2017-02-14 09:10:04,522] INFO     0[main] - com.cloudera.enterprise.dbutil.DbCommandExecutor.loadJDBCDriver(DbCommandExecutor.java) - Unable to find JDBC driver for database type: MySQL</code>
<code>[2017-02-14 09:10:04,528]ERROR     6[main] - com.cloudera.enterprise.dbutil.DbCommandExecutor$DbConnectionTestException.logError(DbCommandExecutor.java) - JDBC Driver com.mysql.jdbc.Driver not found.</code>

**Comment by mfernest** [comment]
You have not yet installed the JDBC connector that Cloudera Manager needs to reach the database. You may have missed this step in the documentation.

**Comment by bagusaptopo** [comment]
Hi Ernest, I've already installed the JDBC connector, I've downloaded from MySQL website using this link: <a href="https://dev.mysql.com/downloads/file/?id=465643">https://dev.mysql.com/downloads/file/?id=465643</a>

After that, I put the jar file in /usr/share/java/, is that correct?

**Comment by mfernest** [comment]
That's correct. The JAR file must also be named correctly so the Cloudera Manager process discovers it. See the file `/etc/default/cloudera-scm-server`.

**Comment by bagusaptopo** [comment]
Thanks for your input, Michael. The package file name for mysql connector is different with the value in cloudera-scm-server. It works after I renamed my package file name. Thanks. :)
